src/libs/actions/User.js

```diff
@@ -0,0 +1,12 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+function getBetas() {
+    Onyx.merge(ONYXKEYS.BETAS, []);
+    API.User_GetBetas().then((response) => {
+        Onyx.set(ONYXKEYS.BETAS, response.betas);
+    });
+}
+
+export default getBetas;
```

Comment:
While this is a totally valid style, I think it's good to have all the actions export things consistently. In this case, it would be:

```
export {
    getBetas,
};
```

Reply:
It was complaining about some warning or something I think... will try it out and report back

Reply:
It works in `GeoLocation.js` so maybe you can refer to that.

Reply:
hmmmm, I see that one is disabling the error I am getting, I will do the same here, but maybe we should remove that rule globally?